api/src/test/java/org/pmiops/workbench/api/WorkspacesControllerTest.java

```diff
@@ -956,20 +955,16 @@ public void testNoteBookList() throws Exception {
     when(fireCloudService.getWorkspace("mockProjectName", "mockWorkspaceName")).thenReturn(
         fcResponse
     );
+    Blob mockBlob = mock(Blob.class);
+    Blob mockBlob1 = mock(Blob.class);
+    when(mockBlob.getName()).thenReturn("notebook/mockFile.ipynb");
+    when(mockBlob1.getName()).thenReturn("notebook/mockFile.text");
+    List<Blob> blobList = ImmutableList.of(mockBlob, mockBlob1);
     when(fireCloudService.getWorkspace("mockProject", "mockWorkspace")).thenThrow(new ApiException());
-    List<String> blobDetailList =
-        ImmutableList.of("notebook/File.ipynb", "notebook/File1.txt", "config/File1.txt");
-    when(cloudStorageService.getBucketFileList("MockBucketName", "notebook"))
-        .thenReturn(blobDetailList);
+    when(cloudStorageService.getBucketFileList("MockBucketName", "notebook")).thenReturn(blobList);
     // Will return 1 entry as only python files in notebook folder are return
     List<FileDetail> result = workspacesController
         .getNoteBookList("mockProjectName", "mockWorkspaceName").getBody();
     assertEquals(result.size(), 1);
-    try {
```

Comment:
Sorry I may have been unclear - I think the test was good, I just do see any reason to do an `assertTrue(true)` specifically.